assets/js/atomic/blocks/product/button/edit.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * External dependencies
+ */
+import { Disabled } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import Block from './block';
+
+export default ( { attributes } ) => {
+	return (
+		<Disabled>
+			<Block { ...attributes } />
```

Comment:
Passing of attributes here is temporary; when we implement context there will be no need to pass attribute data.